docker-build/docker-actions/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:16.17.1-buster
```

Comment:
Maybe call the dir for it more clear - not `docker-actions` but maybe something `unit-test-runner`

Reply:
Please make sure that this docker is part of dependabot and will be update at same time as default dockerfile